src/JMS/Serializer/Metadata/ClassMetadata.php

```diff
@@ -34,6 +34,7 @@ class ClassMetadata extends MergeableClassMetadata
     const ACCESSOR_ORDER_UNDEFINED = 'undefined';
     const ACCESSOR_ORDER_ALPHABETICAL = 'alphabetical';
     const ACCESSOR_ORDER_CUSTOM = 'custom';
+    const COUNT_PARAMS = 16;
```

Comment:
I do not want backward compatible serialization. People have to clear caches between deployments.